portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,100 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/** Places I have lived. */
+var placesLived = [
+  ['Evanston', 42.0451, -87.6877, '2000-2002', 'I was born in outside Chicago at Northwestern Medical Center.'],
```

Comment:
[Optional] This could probably be better organized by using a class but it also adds some overhead.  If we could use [records](https://dzone.com/articles/a-first-look-at-records-in-java-14) that would be best but I don't think it's supported.  I'll leave it up to you.